app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -46,6 +54,15 @@ class SearchLocationFragment : Fragment() {
             rootView.currentLocation.visibility = View.GONE
         })
 
+        rootView.voice_button.setOnClickListener {
+            Utils.hideSoftKeyboard(rootView.context, rootView)
+            checkAudioPermission()
+            if (ContextCompat.checkSelfPermission(requireContext(), Manifest.permission.RECORD_AUDIO)
```

Comment:
Why you are checking again if `checkAudioPermission()` method is already there?

Reply:
Removed extra code.